lib/src/executables/intl_sorting.dart

```diff
@@ -0,0 +1,407 @@
+// Copyright 2021 Workiva Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import 'dart:io';
+import 'package:analyzer/dart/analysis/utilities.dart';
+import 'package:analyzer/dart/ast/ast.dart';
+import 'package:args/args.dart';
+import 'package:codemod/codemod.dart';
+import 'package:file/file.dart';
+import 'package:file/local.dart';
+import 'package:glob/glob.dart';
+import 'package:glob/list_local_fs.dart';
+import 'package:logging/logging.dart';
+import 'package:over_react_codemod/src/intl_suggestors/intl_configs_migrator.dart';
+import 'package:over_react_codemod/src/intl_suggestors/intl_importer.dart';
+import 'package:over_react_codemod/src/intl_suggestors/intl_messages.dart';
+import 'package:over_react_codemod/src/intl_suggestors/intl_migrator.dart';
+import 'package:over_react_codemod/src/util/logging.dart';
+import 'package:over_react_codemod/src/util/package_util.dart';
+import 'package:path/path.dart' as p;
+
+import '../util.dart';
+
+typedef Migrator = Stream<Patch> Function(FileContext);
+
+final _log = Logger('orcm.intl_message_migration');
+
+const _verboseFlag = 'verbose';
+const _yesToAllFlag = 'yes-to-all';
+const _failOnChangesFlag = 'fail-on-changes';
+const _stderrAssumeTtyFlag = 'stderr-assume-tty';
+const _migrateConstants = 'migrate-constants';
+const _migrateComponents = 'migrate-components';
+const _migrateContextMenus = 'migrate-context-menus';
+const _pruneUnused = 'prune-unused';
+const _noMigrate = 'no-migrate';
+
+const _allCodemodFlags = {
+  _verboseFlag,
+  _yesToAllFlag,
+  _failOnChangesFlag,
+  _stderrAssumeTtyFlag,
+};
+
+
+final FileSystem fs = const LocalFileSystem();
+
+
+final parser = ArgParser()
+  ..addFlag(
+    'help',
+    abbr: 'h',
+    negatable: false,
+    help: 'Prints this help output.',
+  )
```

Comment:
I would delete as many flags as possible. -h is probably needed, and maybe -v, but maybe not.